Content.Client/Atmos/EntitySystems/AtmosDebugOverlaySystem.cs

```diff
@@ -99,6 +103,7 @@ internal enum AtmosDebugOverlayMode : byte
     {
         TotalMoles,
         GasMoles,
-        Temperature
+        Temperature,
+        Everything
```

Comment:
Better off just making them flags.

Vera's most familiar with the atmosdebugoverlay anyway so she should review it.